contracts/strategies/ltip-hedgey-governor/LTIPHedgeyGovernorStrategy.sol

```diff
@@ -0,0 +1,205 @@
+    /// ======= External/Custom =======
+    /// ===============================
+
+    /// @notice Update the block number to get voting balances from the Governor contract
+    /// @param _timepoint The new block number
+    function setTimepoint(uint256 _timepoint) external onlyPoolManager(msg.sender) {
+        if (_timepoint == 0) revert INVALID();
+        timepoint = _timepoint;
+        emit TimepointUpdated(msg.sender, _timepoint);
+    }
+
+    /// @notice Revokes allocated votes from a recipient
+    /// @param _recipientId The recipient to revoke votes from
+    /// @param _votes The number of votes to revoke
+    function revokeVotes(address _recipientId, uint256 _votes) external nonReentrant {
+        // Will revert if the updated balances would underflow
+        uint256 _votesCastedFor = votesCastedFor[msg.sender][_recipientId] - _votes;
+        uint256 _updatedVotesCasted = votesCasted[msg.sender] - _votes;
+        uint256 _updatedVotes = votes[_recipientId] - _votes;
+
+        // Update the votes
+        votes[_recipientId] = _updatedVotes;
+        votesCasted[msg.sender] = _updatedVotesCasted;
+        votesCastedFor[msg.sender][_recipientId] = _votesCastedFor;
+
+        emit VotesRevoked(_recipientId);
+
+        if (votes[_recipientId] < votingThreshold) {
+            emit AllocationRevoked(_recipientId, msg.sender);
+        }
+    }
+
+    /// ====================================
+    /// ============ Internal ==============
+    /// ====================================
+
+    /// @notice Allocate (delegated) voting weight to a recipient. In the Governor strategy, we check the votes at a specific block
+    /// @dev '_sender' must have a balance > 0 of delegate tokens
+    /// @param _data The data to be decoded
+    /// @param _sender The sender of the allocation
+    function _allocate(bytes memory _data, address _sender)
+        internal
+        virtual
+        override
+        nonReentrant
+        onlyActiveAllocation
+    {
+        // Decode the '_data'
+        (address recipientId, uint256 _votes) = abi.decode(_data, (address, uint256));
+        Recipient storage recipient = _recipients[recipientId];
+
+        if (recipient.recipientStatus != Status.Accepted) revert RECIPIENT_NOT_ACCEPTED();
+
+        uint256 _votingPower = IGovernor(governorContract).getVotes(_sender, timepoint);
+        uint256 _votesCasted = votesCasted[_sender];
+
+        if (_votingPower == 0) revert VOTING_WEIGHT_ZERO();
+        if (_votesCasted + _votes > _votingPower) revert VOTING_EXCEEDS_WEIGHT();
+
+        // Increment the votes for the recipient
+        votes[recipientId] += _votes;
+        votesCasted[_sender] += _votes;
+        votesCastedFor[_sender][recipientId] += _votes;
+
+        // Emit the event
+        emit Voted(recipientId, _sender);
+
+        if (votes[recipientId] >= votingThreshold) {
+            emit Allocated(recipientId, recipient.allocationAmount, allo.getPool(poolId).token, address(0));
```

Comment:
Why using `address(0)` instead of `_sender`?

Reply:
Good catch, also fixed it in the LTIPSimpleStrategy